powerdns/tests/test_auto_ptr.py

```diff
@@ -89,6 +89,19 @@ def test_default_ptr_created(self):
             owner=self.user,
         )
 
+    def test_ptr_gets_service_from_record(self):
+        record = RecordFactory(
+            domain=self.ptr_domain,
+            type='A',
+            name='site.example.com',
+            content='192.168.1.1',
+            owner=self.user,
+        )
+        Domain.objects.get(name='1.168.192.in-addr.arpa')
+
+        ptr = get_ptr_obj(record.content, record.name)
+        self.assertEqual(record.service, ptr.service)
```

Comment:
i'd also check if service is not None :) (just for case if anyone change factory and somehow it will not assign service anymore and both record's and ptr's service is None)